lib/audio/client/audio-ui.js

```diff
@@ -26,27 +25,27 @@ class AudioUi
         if(!this.audioManager.categories){
             return;
         }
-        let audioSettingsTemplate = this.uiScene.cache.html.get('audio');
-        let audioCategoryTemplate = this.uiScene.cache.html.get('audio-category');
+        let audioSettingsTemplate = this.uiSceneManager.uiSceneDriver.getCacheHtml('audio');
+        let audioCategoryTemplate = this.uiSceneManager.uiSceneDriver.getCacheHtml('audio-category');
         let audioSettingsContent = this.prepareAudioSettingsContent(audioCategoryTemplate, audioSettingsTemplate);
-        this.gameManager.gameDom.appendToElement('#settings-dynamic', audioSettingsContent);
-        let audioSettingInputs = this.gameManager.gameDom.getElements('.audio-setting');
+        this.uiSceneManager.gameDom.appendToElement('#settings-dynamic', audioSettingsContent);
+        let audioSettingInputs = this.uiSceneManager.gameDom.getElements('.audio-setting');
         if(0 === audioSettingInputs.length){
             return false;
         }
         for(let settingInput of audioSettingInputs){
             settingInput.addEventListener('click', async (event) => {
                 await this.audioManager.setAudio(event.target.dataset['categoryKey'], settingInput.checked);
-                this.sendAudioUpdate(settingInput.value, settingInput.checked, this.gameManager.activeRoomEvents);
-                this.sceneAudioPlayer.playSceneAudio(this.audioManager, this.gameManager.getActiveScene());
+                this.sendAudioUpdate(settingInput.value, settingInput.checked, this.uiSceneManager.getActiveRoomEventsCallback());
```

Comment:
The logic behind it is to prevent sending a whole object like roomevents o gameManager through all the code. It's more efficient and less error prone to pass only what's needed inside classes and methods than passing a whole object.

Reply:
Checked out how colyseus readme.md handles contributors, it uses a tool called [all-contributors](https://allcontributors.org/) that can be implemented using a CLI-tool or a GitHub bot. I don't know if the tool has to be installed in my fork or in the main repo and how (CLI is easy, but I've never installed a bot here).